### PR TITLE
add Windows package and update README

### PR DESCRIPTION
**Pull request by 96jonesa**
Adds Windows tar.gz and updates README guides to explain how to use on Windows

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,24 +32,37 @@ AutoBought is an automated software designed to track the availability of and au
...
-#### OSX / Windows
+#### OSX
 
 Work in progress
 
+#### Windows
+
+1. Clone or download this repository
+2. unzip `AutoBoughtPackageLinux.tar.gz`
+3. unzip `AutoBoughtPackageLinux.tar`
```
Comment (maryamsaleh) [inline README.md:46]: Is this supposed to say `AutoBoughtPackageWindows`?

**Review by maryamsaleh — CHANGES_REQUESTED** [review]
Everything looks good but it looks like there might be a typo in lines 45-46 of the README. If not let me know and I can approve and merge

File: README.md (hunk 1 above)
Comment (96jonesa) [reply README.md:46]: o ye ill fix

**Review by maryamsaleh — APPROVED** [review]
Looks good now

**Pull request merged**